osh/hub/settings_local.py

```diff
@@ -130,4 +130,9 @@
 # E-mail address where notifications about failed/cancelled tasks are sent
 DEVEL_EMAIL_ADDRESS = "no-reply@example.org"
 
+# TODO: What should we use here if we want to deploy across multiple clouds?
+RESALLOC_WORKER_SSH_PRIVATE_KEY = "id_rsa"
+OSH_HUB_URL = "http://ec2-54-202-223-253.us-west-2.compute.amazonaws.com:8000/xmlrpc"
+ENABLE_THROWAWAY_WORKERS = True
```

Comment:
Shall we name this setting differently? `OSH_HUB_URL` and `RESALLOC_WORKER_SSH_PRIVATE_KEY` are only used if this setting is enabled.

Reply:
Do we need to manually configure `OSH_HUB_URL`?  Cannot the information be obtained from Django and/or Apache?  Could we at least use some automagically obtained value as a default to cover  the most common cases?

What about renaming `ENABLE_THROWAWAY_WORKERS` to `ENABLE_SINGLE_USE_WORKERS`?

Reply:
> Do we need to manually configure OSH_HUB_URL? Cannot the information be obtained from Django and/or Apache? 

> Could we at least use some automagically obtained value as a default to cover the most common cases?

The hostname should be reachable to the worker and is only used when resalloc is enabled. I am not aware what value may be suitable for such case. We can probably rename this setting to `RESALLOC_OSH_HUB_URL` and leave a blank value as default.

Reply:
Yes, we can certainly do that now and eventually figure out a way how to query/guess the correct value when `RESALLOC_OSH_HUB_URL` is not specified in the config file.

Reply:
On a second note, shall we use the term `RESALLOC` at all in the OpenScanHub source code? We might just use the term `SINGLE_USE_WORKERS` everywhere, just in case we switch to some other backend for running virtual machines dynamically.

Reply:
We should probably rename this script to `osh-worker-manager` (or something similar). And create a subpackage with same name, if needed.